### PR TITLE
Learning rate

### PR DESCRIPTION
**Pull request by SallyElHajjar**
# Use an exponential decay schedule for a smoother learning rate reduction.

### CODE REVIEW

File: usl_models/usl_models/atmo_ml/model.py [hunk 1]
```diff
@@ -153,9 +153,17 @@ def from_checkpoint(cls, artifact_uri: str, **kwargs) -> "AtmoModel":
 
     def _build_model(self) -> keras.Model:
         """Creates the correct internal (Keras) model architecture."""
+        # Use an exponential decay schedule for a smoother learning rate reduction.
+        lr_schedule = tf.keras.optimizers.schedules.ExponentialDecay(
```
Comment (Katsutoshii) [inline usl_models/usl_models/atmo_ml/model.py:157]: Can you move this under the default value for the model params?

**Comment by Katsutoshii** [comment]
Can you show a tensorboard plot comparing the training run before and after this change?

File: usl_models/usl_models/atmo_ml/model.py (hunk 1 above)
Comment (SallyElHajjar) [reply usl_models/usl_models/atmo_ml/model.py:164]: Done

**Comment by SallyElHajjar** [comment]
WITHOUTDECAY

**Comment by Katsutoshii** [comment]
What's the change in MSE @1000 epochs?

**Comment by SallyElHajjar** [comment]
@Katsutoshii wasn't able to get the plots because of tensorboard strange issue. Didn't found the plot after I made the change (it looks like one point. ThAT's why I have just reported these plots. Is that necessarily to repeat or these plots work?

**Comment by Katsutoshii** [comment]
Let's repeat the run, I'd like to have a loss number we can compare against previous runs.
